dashboard_schema.json

```diff
@@ -199,7 +199,7 @@
                 "type": "string"
               },
               "resolution": {
-                "type": "integer"
+                "enum": [0.5, 1, 2, 3, 4, 5]
```

Comment:
`0.5` and `1` are included because the graph resolution value, if set on an individual graph, can be anything `1-5`. It isn't until the graph is refreshed that the resolution will be set to `0.5`